### PR TITLE
background color nav action added breadcrumb

### PR DESCRIPTION
**Pull request by alexusernamebruh**
Light theme
image

-

Dark theme
image

-
I couldn't find how to change the light and dark theme from this icon(I googled for like 30 minutes and found nothing on how to do it) so I created an control instead
image
image

### CODE REVIEW

**Comment by ssysm** [comment]
single quote please

File: src/components/Breadcrumbs/index.stories.tsx [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
 
-import { Breadcrumbs, BreadcrumbsProps } from './';
+import { Breadcrumbs, BreadcrumbsProps } from "./";
 
 // More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 export default {
-  title: 'Elements/Breadcrumbs',
+  title: "Elements/Breadcrumbs",
```
Comment (ssysm) [inline src/components/Breadcrumbs/index.stories.tsx:8]: single

**Review by ssysm — CHANGES_REQUESTED** [review]

File: src/components/Breadcrumbs/index.stories.tsx [hunk 2]
```diff
@@ -1,35 +1,33 @@
...
     {
-      title: 'Title',
-      href: '/stories',
+      title: "Title",
+      href: "/stories",
     },
     {
-      title: 'Title',
-      href: '/stories',
+      title: "Title",
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.stories.tsx:23]: still has double quotes

File: src/components/Breadcrumbs/index.tsx [hunk 3]
```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
...
+let background = 'light';
+const DarkMode =
+  'bg-gray-900  flex px-2 py-2 border rounded-lg  hover:border-gray-600';
 export const Breadcrumbs: React.FC<BreadcrumbsProps> = ({
   links,
   customLink = {
     component: (props: any) => <a {...props} />,
     hrefProperty: 'href',
   },
+  backgroundColor = background,
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: backgroundColor = 'light'

File: src/components/Breadcrumbs/index.tsx [hunk 4]
```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
     component: any;
     hrefProperty: string;
   };
+  backgroundColor?: 'light' | 'dark';
   links: ILink[];
 }
 
+let background = 'light';
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: remove

File: src/components/Breadcrumbs/index.tsx [hunk 5]
```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
...
+const DarkMode =
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: do the dark mode background handling inside the stories file by just adding another ComponentStory

File: src/components/Breadcrumbs/index.tsx [hunk 6]
```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
     component: any;
     hrefProperty: string;
   };
+  backgroundColor?: 'light' | 'dark';
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: darkMode?: boolean

File: src/components/Breadcrumbs/index.tsx [hunk 7]
```diff
@@ -14,61 +14,120 @@ export interface BreadcrumbsProps {
...
-          </div>
-        </li>
-        {links.slice(1).map((page, i) => (
-          <li key={i} title={page.title}>
-            <div className='flex items-center'>
-              <ChevronRightIcon
-                className='flex-shrink-0 w-5 h-5 text-gray-300'
-                aria-hidden='true'
-              />
+  if (backgroundColor === 'light') {
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: instead of doing this if statement, just do ternary for the classnames checking for darkMode

**Review by RichardX366 — CHANGES_REQUESTED** [review]

**Comment by mxdi9i7** [comment]
Where is the code that actually adds the background?

**Pull request closed without merging**